invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html

```diff
@@ -0,0 +1,257 @@
+    </p>
+    <ul>
+      <li>
+        <code>[2017 TO 2018}</code> ist aufgrund der Datumserweiterung
+        und der exklusiven Obergrenze äquivalent zu
+        <code>[2017-01-01 TO 2017-12-31]</code>.
+      </li>
+    </ul>
+    <p>Beispiele für andere Bereiche:</p>
+    <ul>
+      <li>
+        <code>metadata.publication_date:{* TO 2017-01-01}</code>: Alle Tage bis 2017.
+      </li>
+      <li>
+        <code>metadata.publication_date:[2017-01-01 TO *]</code>: Alle Tage von 2017.
+      </li>
+    </ul>
+    <h3>Ranking/Sortierung</h3>
+    <p>
+      Standardmäßig werden alle Suchen nach einem internen Ranking-Algorithmus sortiert,
+      der jede Übereinstimmung mit Ihrer Abfrage bewertet. Sowohl in der Benutzeroberfläche
+      als auch in der REST-API ist es möglich, die Ergebnisse zu sortieren nach:
+    </p>
+    <ul>
+      <li>Aktuellste</li>
+      <li>Beste Übereinstimmung</li>
+    </ul>
+    <h3>Reguläre Ausdrücke</h3>
+    <p>
+      Reguläre Ausdrücke sind eine mächtige Sprache für den Mustervergleich, mit der
+      man nach bestimmten Mustern in einem Feld suchen kann. Wenn wir zum Beispiel
+      alle Datensätze mit dem DOI-Präfix 10.5281 finden wollen, können wir eine Suche mit
+      regulären Ausdrücken verwenden:
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=metadata.subjects.identifier:%2F03yrm5c2%5B1%2C6%5D%2F"
+        ><code>metadata.subjects.identifier:/03yrm5c2[1,6]/</code></a
+      >
+    </p>
+    <p>
+      Vorsicht, der reguläre Ausdruck muss mit dem <em>gesamten</em> Feldwert übereinstimmen.
+      Weitere Einzelheiten finden Sie in der
+      <a
+        href="https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html#regexp-syntax"
+        >Syntax des regulären Ausdrucks</a
+      >
+      .
+    </p>
+    <h3>Fehlende Werte</h3>
+    <p>
+      Es ist möglich, mit den Feldnamen <code>_exists_</code> und
+      <code>_missing_</code> nach Datensätzen zu suchen,
+      in denen entweder ein Wert fehlt oder die einen Wert in einem bestimmten Feld haben.
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=_missing_:metadata.additional_titles"
+        ><code>_missing_:metadata.additional_titles</code></a
+      >
+      (alle Datensätze ohne metadata.additional_titles)
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=_exists_:metadata.creators"
+        ><code>_exists_:metadata.creators</code></a
+      >
+      (alle Datensätze mit metadata.creators)
+    </p>
+    <h3>Erweiterte Konzepte</h3>
+    <h4>Boosten</h4>
+    <p>
+      Sie können den Boost-Operator <code>^</code> verwenden, wenn ein Begriff relevanter ist als ein
+      anderer. Sie können z. B. nach allen Datensätzen suchen, die den Ausdruck
+      <em>open science</em> entweder <em>im title</em> oder im
```

Comment:
Should the 'im' in 'im title' be emphazized? The next one (im description) is not.